test_scripts/Defects/8_1/3859_CreateInteractionChoiceSet_infor_in_response_during_erroneous_response.lua

```diff
@@ -0,0 +1,78 @@
+---------------------------------------------------------------------------------------------------
+-- Issue: https://github.com/smartdevicelink/sdl_core/issues/3859
+---------------------------------------------------------------------------------------------------
+-- Description: Check that SDL transfers the error message from the HMI response to the mobile app
+
+-- Steps:
+-- 1. HMI and SDL are started
+-- 2. Mobile app is registered and activated
+-- 3. Mobile app requests CreateInteractionChoiceSet RPC
+-- 4. SDL sends VR.AddCommand(type = "Choice") request to the HMI
+-- 5. HMI responds with erroneous code and error message to the VR.AddCommand(type = "Choice") request
+-- SDL does:
+-- 1. process the CreateInteractionChoiceSet response from HMI
+-- 2. send the CreateInteractionChoiceSet response to the mobile app with received error code and message
+---------------------------------------------------------------------------------------------------
+
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('user_modules/sequences/actions')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+local requestParams = {
+  interactionChoiceSetID = 1001,
+  choiceSet = {
+    {
+      choiceID = 1001,
+      menuName ="Choice1001",
+      vrCommands = {
+        "Choice1001"
+      }
+    }
+  }
+}
+local errorCodes = {
+  "DISALLOWED",
+  "REJECTED",
+  "ABORTED",
+  "IGNORED",
+  "IN_USE",
+  "TIMED_OUT",
+  "INVALID_DATA",
+  "CHAR_LIMIT_EXCEEDED",
+  "INVALID_ID",
+  "DUPLICATE_NAME",
+  "GENERIC_ERROR"
+}
```

Comment:
I propose to extend coverage for errorcode:
 "UNSUPPORTED_REQUEST",
  "USER_DISALLOWED",
  "READ_ONLY"